_includes/manuals/1.8/4.1.1_authentication.md

```diff
@@ -33,6 +33,8 @@ On Red Hat based OSes:
 {% highlight bash %}
 cp example.crt /etc/pki/tls/certs/
 ln -s example.crt /etc/pki/tls/certs/$(openssl x509 -noout -hash -in /etc/pki/tls/certs/example.crt).0
+unlink /etc/pki/tls/certs/ca-bundle.crt
```

Comment:
I wouldn't recommend this, that's your system trust store and it might break other applications.